numpy/ma/core.py

```diff
@@ -26,11 +26,7 @@
 import textwrap
 import re
 from functools import reduce
-
-if sys.version_info[0] >= 3:
```

Comment:
remove `import sys` on L23

Reply:
still needed

Reply:
Is the `if sys.version_info.major < 3` block on L3894 remaining for `def __unicode__(self)`?

Reply:
Yes, these are the "hard" cases for cleaning up sys.version, I have another PR that will follow this one cleaning up the easy cases.